Xwt.Gtk/Xwt.GtkBackend/GtkAlertDialog.cs

```diff
@@ -123,7 +123,7 @@ public GtkAlertDialog (ApplicationContext actx, MessageDescription message)
 			}
 			
 			if (message.AllowApplyToAll) {
-				Gtk.CheckButton check = new Gtk.CheckButton ("Apply to all");
+				Gtk.CheckButton check = new Gtk.CheckButton (Application.TranslationCatalog.GetString("Apply to all"));
```

Comment:
Oh thanks, and sorry! Fixed: https://github.com/mono/xwt/commit/f5f9a7449d68f4362aecf0ed800d336ca50fe0fd